sql/bgw_scheduler.sql

```diff
@@ -73,6 +73,16 @@ AS '@MODULE_PATHNAME@', 'ts_policy_compression_proc'
 LANGUAGE C;
 
 /* continuous aggregates policy */
+CREATE OR REPLACE FUNCTION add_refresh_continuous_aggregate_policy(continuous_aggregate REGCLASS, start_interval "any",  end_interval "any" , schedule_interval INTERVAL, if_not_exists BOOL = false)
+RETURNS INTEGER
+AS '@MODULE_PATHNAME@', 'ts_policy_refresh_cagg_add'
+LANGUAGE C VOLATILE STRICT;
+
+CREATE OR REPLACE FUNCTION remove_refresh_continuous_aggregate_policy(continuous_aggregate REGCLASS, if_not_exists BOOL = false)
+RETURNS INTEGER
```

Comment:
Same question here about the return value. Why integer?

Reply:
fixed.